using-webauthn-mfa-in-command-line.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: default
+title: Using WebAuthn multifactor authentication in command line
+url: /using-webauthn-mfa-in-command-line
+previous: /using-mfa-in-command-line
+next: /using-otp-mfa-in-command-line
+---
+<em class="t-gray">How to use WebAuthn MFA with gem CLI.</em>
+
+Multifactor authentication (MFA) using WebAuthn works by using a
+removable hardware token or touch biometric / facial biometric capabilities built
+into your phone or computer. This is distinct from MFA based on typing or copying
+a code generated by an authentication app or password manager, called OTP. For
+OTP MFA see "[Using OTP multifactor authentication in command line](/ using-otp-mfa-in-command-line)".
+
+When you have enabled WebAuthn MFA, and your MFA level is _UI and API_, we will ask
+you to perform authentication for `gem signin`, `gem push`, `gem owner --add`,
+`gem owner --remove` and `gem yank`. Check "[Setting up WebAuthn multifactor authentication](/setting-up-webauthn-multifactor-authentication)"
+for guidance on setting up WebAuthn MFA.
```

Comment:
This is a bit complicated. By default, if you have a security device setup, it'll be equivalent to `UI and gem signin` (MFA on web and gem signin).

If you have **OTP** setup on `UI and API`, we will ask
you to perform authentication for `gem signin`, `gem push`, `gem owner --add`,
`gem owner --remove` and `gem yank`.

Reply:
Should we change these docs or change the behaviour?

Reply:
We're planning to change the behaviour to that after the release since it'll take a considerable amount of effort. 

I think we should change the docs with the behaviour now and add a clause that we are still working to make WebAuthn be a drop-in replacement for time-based one time passwords (TOTP) and users who register a security device will be able to select the appropriate MFA level for their account without needing to set up TOTP based authentication.

Reply:
👍 makes sense to me. I'll change it later this afternoon.